code/hsec-tools/cabal.project

```diff
@@ -1,5 +1,3 @@
-tested-with: GHC==9.2.5
```

Comment:
I removed this line because my cabal executable objected to it as out of place. If I've misunderstood we could re-add it

Reply:
That's fine, I had the same warning too, but didn't get around to removing it yet.

Reply:
Also, feel free to extract this change and submit as an independent PR!